Resources/Maps/packedstation.yml

```diff
@@ -18119,13 +18119,12 @@ entities:
   - pos: -4.3740964,-12.360607
     parent: 0
     type: Transform
+  - cellSlot:
+      startingItem: PowerCellSmallStandard
+      name: power-cell-slot-component-slot-name-default
+    type: PowerCellSlot
```

Comment:
This is a known issue that is not related to this PR.

Reply:
It also involved bar signs saving I think but I removed those. Wasn't sure if removing all those power cell lines would break something on the map like emergency lights. Thoughts on removing?